toplevel/dune

```diff
@@ -9,5 +9,6 @@
 
 (rule
  (targets g_toplevel.ml)
+ (mode promote-until-clean)
```

Comment:
This also seems like a mess to me, it won't be long before people do commit these files by mistake.

Instead Coq should switch to a supported pre-processing solution in the OCaml ecosystem; however we are not the only ones having the problem so I guess a bit more discussion in the next Dune meeting will be good.

Reply:
The fact that there is a rule `g_*.ml` in `.gitignore` should prevent this.